src/main/java/org/mskcc/oncokb/transcript/service/dto/UserDTO.java

```diff
@@ -1,24 +1,71 @@
 package org.mskcc.oncokb.transcript.service.dto;
 
+import java.time.Instant;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.validation.constraints.*;
+import org.mskcc.oncokb.transcript.config.Constants;
+import org.mskcc.oncokb.transcript.domain.Authority;
 import org.mskcc.oncokb.transcript.domain.User;
 
 /**
- * A DTO representing a user, with only the public attributes.
+ * A DTO representing a user, with their authorities.
  */
 public class UserDTO {
 
     private Long id;
 
+    @NotBlank
+    @Pattern(regexp = Constants.LOGIN_REGEX)
+    @Size(min = 1, max = 50)
     private String login;
 
+    @Size(max = 50)
+    private String firstName;
+
+    @Size(max = 50)
+    private String lastName;
+
+    @Email
+    @Size(min = 5, max = 254)
```

Comment:
We should double check the Size limits vs liquibase definition. This email here allows 254, but liquibase has it as 191. Could you go over all limitations and see what causes the discrepancy ?

Reply:
This seems to be related to [issue from jhipster](https://github.com/jhipster/generator-jhipster/issues/8604). 
Since we're using utf8, then the max length for our unique email field can be 255 characters instead of 191. Should be okay to increase the liquibase constraint to 255.